src/librustc_driver/target_features.rs

```diff
@@ -24,64 +24,14 @@ use syntax::parse::token::intern_and_get_ident as intern;
 /// SubtargetInfo, which knows all this information, but the ways to
 /// query them do not seem to be public.
 pub fn add_configuration(cfg: &mut ast::CrateConfig, sess: &Session) {
-    let tf = InternedString::new("target_feature");
-    macro_rules! fillout {
-        ($($func: ident, $name: expr;)*) => {{
-            $(if $func(sess) {
-                cfg.push(attr::mk_name_value_item_str(tf.clone(), intern($name)))
-            })*
-        }}
-    }
-    fillout! {
-        has_sse, "sse";
-        has_sse2, "sse2";
-        has_sse3, "sse3";
-        has_ssse3, "ssse3";
-        has_sse41, "sse4.1";
-        has_sse42, "sse4.2";
-        has_avx, "avx";
-        has_avx2, "avx2";
-        has_neon, "neon";
-        has_vfp, "vfp";
-    }
-}
-
+    unsafe { ::rustc_trans::back::write::configure_llvm(sess); }
```

Comment:
I think there's a `Once` guarding this somewhere else, and this probably wants to go that same route rather than running this unconditionally (which can be racy)